package.json

```diff
@@ -81,7 +81,7 @@
     "jest": "^24.9.0",
     "jest-circus": "^24.9.0",
     "jest-static-stubs": "^0.0.1",
-    "lerna": "^3.16.4",
+    "lerna": "3.20.2",
```

Comment:
Missing a hat here `"lerna": "^3.20.2",`

Reply:
I can manually add the hat, but I'm curious what command I should have used.

I tried `yarn upgrade lerna --latest`, which didn't make any changes (I don't understand why)

So, instead I found the latest version of lerna from npm, and used `yarn add lerna@3.16.4`. How do you get the hat in there without having to manually modify package.json?

I think it's good to match the currently established convention, but I'm wondering why the hat is considered best practice? Seems more repeatable to have a specific version specified, seeing as we manually do upgrades.

Reply:
When you upgrade to a specific version, you can indicate it as you would see in the package: `yarn upgrade left-pad@^1.0.0`. TIL you can also pass in caret: https://classic.yarnpkg.com/en/docs/cli/upgrade/#toc-yarn-upgrade-package-latest-l-caret-tilde-exact-pattern

There's a couple of different takes on why caret is best practice. In theory, it allows you to accept future bug fixes and features that shouldn't break things. Of course, sometimes shit happens and something intended to be a bug fix could be a breaking change. Or sometimes what you've written with only work with a specific version of a package. This why some people like to pin their dependencies. 

IMO, lock files do a pretty good job at mitigating against that.

Reply:
Thanks for this!

Reply:
I added the hat - ready for re-review.